.editorconfig

```diff
@@ -1,4 +1,24 @@
-[{*.kt,*.kts}]
+root = true
+
+[*]
+charset = utf-8
+indent_size = 2
+indent_style = space
+insert_final_newline = true
```

Comment:
I see you have removed new line at the end of file, but this say we need to insert it 🙂 
Can we leave end new line?

Reply:
`insert_final_newline` is for inserting new lines. I removed some lines accidentally that may caused by the default IDE configs